src/api/app/models/review.rb

```diff
@@ -273,6 +277,22 @@ def changed_state_at
     self[:changed_state_at] || self[:updated_at]
   end
 
+  def requested_to_user?
```

Comment:
I don't understand the name `requested_to_user` for this method. Could we rename this and the other similar methods to `for_*?`, like `for_user?`? It would read much better.

Reply:
Done